### ISSUE TITLE
 How can I get the KKBox split as described in the paper?

### ISSUE THREAD

**Issue by LeeJunHyun** [havakv/pycox#61]
I really appreciate your commitment to this field.

I  got kkbox dataset by using `kkbox.read_df()`.

The kkbox dataset consists of 2,814,735 instances, so how can I get the same split as described in the paper?

In the paper, there are 1,786,333 train samples, 661,748 test samples, and 198,665 valid samples.
(2,646,746 instances in total)

Thanks.

**Comment by havakv** [comment]
Thank for the kind words! 

You're probably looking for `kkbox_v1.read_df()` which is the dataset used in "Time-to-event prediction with neural networks and Cox regression".

The dataset in `kkbox.read_df()` is used in the paper "The Brier Score under Administrative Censoring: Problems and Solutions" and has some improvements compared to `kkbox_v1` in addition to administrative censoring times.

**Comment by havakv** [comment]
From [the read_df docs](https://github.com/havakv/pycox/blob/67d213b96d538a455b9b3d044c9aaed401ab4c93/pycox/datasets/from_kkbox.py#L83) you can see that `kkbox_v1.read_df` accepts arguments for training, validation and test set.

**Comment by LeeJunHyun** [comment]
@havakv Thank you so much! Your reply is a great help for me :)

**Comment by havakv** [comment]
Happy to help! I'll close this issue than, and you can reopen it if you don't consider it solved.
